projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -371,7 +391,7 @@ private function prepare_menu_item_url( $url, $parent_slug = '' ) {
 			)
 		) {
 			if (
-				( 'admin.php' !== $parent_file && file_exists( WP_PLUGIN_DIR . "/$parent_file" ) && ! is_dir( WP_PLUGIN_DIR . "/$parent_file" ) ) ||
+				( ( false === $admin_is_parent || null === $admin_is_parent ) && 'admin.php' !== $parent_file && file_exists( WP_PLUGIN_DIR . "/$parent_file" ) && ! is_dir( WP_PLUGIN_DIR . "/$parent_file" ) ) ||
```

Comment:
If the default for `$admin_is_parent` was `false`, the `null` check here could be omitted.

Reply:
Yes, the null check is redundant now, thanks!